content/source/docs/enterprise/private/gcp-setup-guide.html.md

```diff
@@ -0,0 +1,353 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Installation (Installer) - GCP Setup Guide"
+sidebar_current: "docs-enterprise2-private-installer-gcp"
+description: |-
+  This document provides recommended practices and a reference architecture for
+  HashiCorp Private Terraform Enterprise (PTFE) implementations on GCP.
+---
+
+# Private Terraform Enterprise GCP Setup Guide
+
+This document provides recommended practices and a reference architecture for
+HashiCorp Private Terraform Enterprise (PTFE) implementations on GCP.
+
+## Required Reading
+
+Prior to making hardware sizing and architectural decisions, read through the
+[installation information available for
+PTFE](https://www.terraform.io/docs/enterprise/private/install-installer.html)
+to familiarise yourself with the application components and architecture.
+Further, read the [reliability and availability
+guidance](https://www.terraform.io/docs/enterprise/private/reliability-availability.html)
+as a primer to understanding the recommendations in this reference
+architecture.
+
+## Infrastructure Requirements
+
+Depending on the chosen [operational
+mode](https://www.terraform.io/docs/enterprise/private/install-installer.html#operational-mode-decision),
+the infrastructure requirements for PTFE range from a single Cloud Compute VM instance
+for demo installations to multiple instances connected to Cloud SQL, Cloud Storage, and an
+external Vault cluster for a stateless production installation.
+
+This reference architecture focuses on the “Production - External
+Services” operational mode.
+
+The following table provides high-level server guidelines. Of particular
+note is the strong recommendation to avoid non-fixed performance CPUs,
+or “Burstable CPU” in GCP terms, such as T-series instances.
```

Comment:
Did GCP adopt instance series naming like AWS?  Do they have an equivalent to `t2.micro`?

Reply:
They have F and G series instances for bursting. I need to update this.

Reply:
Fixed in latest commit